src/lib/optimizer/chunk_statistics/abstract_filter.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <memory>
+
+#include "all_type_variant.hpp"
+#include "types.hpp"
+
+namespace opossum {
+
+/**
+ * Base class for all Filters. A Filter is part of ChunkColumnStatistics and holds
+ * some kind of statistical information about the contents of a column within a chunk.
+ * This information can be used to optimize SQL Queries.
```

Comment:
Minor: SQL ist not relevant anymore on this level of abstraction, just talk about "queries"